specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,498 @@
+
+`public Windows.UI.Xaml.Media.Brush SelectedDisabledBorderBrush { get; set; }`
+
+Gets or sets a brush that provides the border of a selected calendar item while it's disabled.
+
+`public Windows.UI.Xaml.Media.Brush CalendarItemHoverBackground { get; set; }`
+
+Gets or sets a brush that provides the background of a calendar item while the pointer is over it.
+
+`public Windows.UI.Xaml.Media.Brush CalendarItemPressedBackground { get; set; }`
+
+Gets or sets a brush that provides the background of a calendar item while it's pressed.
+
+`public Windows.UI.Xaml.Media.Brush CalendarItemDisabledBackground { get; set; }`
+
+Gets or sets a brush that provides the background of a calendar item while it's disabled.
+
+`public Windows.UI.Xaml.Media.Brush OutOfScopeHoverForeground { get; set; }`
+
+Gets or sets a brush that provides the foreground of calendar items that are outside the current scope (month, year, or decade) while the pointer is over them.
+
+`public Windows.UI.Xaml.Media.Brush OutOfScopePressedForeground { get; set; }`
+
+Gets or sets a brush that provides the foreground of calendar items that are outside the current scope (month, year, or decade) while they are pressed.
+
+`public Windows.UI.Xaml.Media.Brush TodayBackground { get; set; }`
+
+Gets or sets a brush that provides the background of the calendar item for the current date.
+
+`public Windows.UI.Xaml.Media.Brush BlackoutBackground { get; set; }`
+
+Gets or sets a brush that provides the background of calendar items while they are blacked out.
+
+`public Windows.UI.Xaml.Media.Brush TodayBlackoutBackground { get; set; }`
+
+Gets or sets a brush that provides the background of the calendar item for the current date while it's blacked out.
+
+`public Windows.UI.Xaml.Media.Brush TodayBlackoutForeground { get; set; }`
+
+Gets or sets a brush that provides the foreground of the calendar item for the current date while it's blacked out.
+
+`public Windows.UI.Xaml.Media.Brush TodayHoverBackground { get; set; }`
+
+Gets or sets a brush that provides the background of the calendar item for the current date while the pointer is over it.
+
+`public Windows.UI.Xaml.Media.Brush TodayPressedBackground { get; set; }`
+
+Gets or sets a brush that provides the background of the calendar item for the current date while it's pressed.
+
+`public Windows.UI.Xaml.Media.Brush TodayDisabledBackground { get; set; }`
+
+Gets or sets a brush that provides the background of the calendar item for the current date while it's disabled.
+
+`public Windows.UI.Xaml.Media.Brush TodayDisabledForeground { get; set; }`
+
+Gets or sets a brush that provides the foreground of the calendar item for the current date while it's disabled.
+
+`public Windows.UI.Xaml.Media.Brush TodaySelectedInnerBorderBrush { get; set; }`
+
+Gets or sets a brush that provides the border of the calendar item for the current date while it's selected.
+
+
+# API Details
```

Comment:
Will there also be a MUXC version, or only WUXC?

Reply:
These OS changes will be supplemented by WinUI 2.6 style updates for the CalendarView control. The new WinUI 2.6 style will be the only one using the new properties, via the boolean CalendarViewBaseItemRoundedChromeEnabled resource.

OS Xaml has that boolean set to False by default and does not use the new rendering & properties. 
WinUI 2.6 sets the resource to True and that triggers the use of the new rendering & properties.
We did the same for the new ListViewItemPresenter properties (using the boolean resource called 'ListViewBaseItemRoundedChromeEnabled').